prisma/schema.prisma

```diff
@@ -59,6 +59,18 @@ model Challenge {
   certificate   Certificate @relation(fields: [certificateId], references: [id], onDelete: Cascade)
 }
 
+model SystemState {
+  // As the following type is unique, enum, requored and only one option
+  // it enforces that there is only one column in the table
```

Comment:
Do you mean `only one row in the table`?  We could add other global state here (i.e., columns) if we needed to, no? Or do I misunderstand?

Reply:
Yes, you are correct!